index.bs

```diff
@@ -23,46 +23,56 @@ Test Suite: https://github.com/w3c/web-platform-tests/tree/master/IndexedDB
 spec: html; urlPrefix: https://html.spec.whatwg.org/multipage/
```

Comment:
Yeah I'm fixing the plural ones. Usually you should be able to use for="" or spec="" or lt="" to resolve the rest without using link-defaults.

Worker is an interface, not a term, so that's probably why it's confused there.